python3

```diff
@@ -0,0 +1 @@
+bash
```

Comment:
`python3` is a symbolic link to `bash`. This isn't required. This just demonstrates how one can run a different command in the container simply by using different names for the same script.